docs/index.rst

```diff
@@ -112,27 +112,19 @@ Review this information at your leisure.
 
    Architecture and engineering Confluence information hub <https://openedx.atlassian.net/wiki/spaces/AC/overview>
    Open edX glossary <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/glossary.html>
-   Supported browsers <universal_stub>
+   technical_reference/browser_support
    technical_reference/named_releases
 
 
 
 Information for other user groups
 =================================
 
-* :doc:`Course authoring teams <universal_stub>`
-* :doc:`System administrators <universal_stub>`
-* :doc:`Researchers <universal_stub>`
-* :doc:`Open edX partners <universal_stub>`
-* :doc:`Learners <universal_stub>`
+If your tasks include system administration, course authoring, or anything outside of software development, review the options on `Documentation for edx.org and the Open edX community <http://docs.edx.org>`_.
 
 
 .. toctree::
    :hidden:
    :caption: Information for other user groups
 
-   Course authoring teams <universal_stub>
-   System administrators <universal_stub>
-   Researchers <universal_stub>
-   Open edX partners <universal_stub>
-   Learners <universal_stub>
+   Documentation for edx.org and the Open edX community <http://docs.edx.org>
```

Comment:
Looks like a mis-formatted reference.

Reply:
For the toctree, this formatting is correct. I just ran make docs locally and the link is rendering correctly in the TOC. Unfortunately I can't insert a screenshot here as far as I know.